include/zephyr/net/http/service.h

```diff
@@ -88,6 +100,31 @@ struct http_service_desc {
 #define HTTP_SERVICE_DEFINE_EMPTY(_name, _host, _port, _concurrent, _backlog, _detail)             \
 	__z_http_service_define(_name, _host, _port, _concurrent, _backlog, _detail, NULL, NULL)
 
+/**
+ * @brief Define an HTTPS service without static resources.
+ *
+ * @note The @p _host parameter must be non-`NULL`. It is used to specify an IP address either in
+ * IPv4 or IPv6 format a fully-qualified hostname or a virtual host.
+ *
+ * @note The @p _port parameter must be non-`NULL`. It points to a location that specifies the port
+ * number to use for the service. If the specified port number is zero, then an ephemeral port
+ * number will be used and the actual port number assigned will be written back to memory. For
+ * ephemeral port numbers, the memory pointed to by @p _port must be writeable.
+ *
+ * @param _name Name of the service.
+ * @param _host IP address or hostname associated with the service.
+ * @param[inout] _port Pointer to port associated with the service.
+ * @param _concurrent Maximum number of concurrent clients.
+ * @param _backlog Maximum number queued connections.
+ * @param _detail Implementation-specific detail associated with the service.
+ * @param _sec_tag_list TLS security tag list used to setup a HTTPS socket.
+ * @param _sec_tag_list_size TLS security tag list size used to setup a HTTPS socket.
+ */
+#define HTTPS_SERVICE_DEFINE_EMPTY(_name, _host, _port, _concurrent, _backlog, _detail,          \
+				   _sec_tag_list, _sec_tag_list_size)                            \
+	__z_http_service_define(_name, _host, _port, _concurrent, _backlog, _detail, NULL, NULL, \
+				_sec_tag_list, _sec_tag_list_size)
```

Comment:
This one appears to be missing the `BUILD_ASSERT` that `HTTPS_SERVICE_DEFINE` has?